app/templates/play/menu/inventory-modal.jade

```diff
@@ -79,6 +79,7 @@
       button#unequip-item-viewed.btn.secret(data-i18n="inventory.unequip")
       button#subscriber-item-viewed.btn.big-font.secret(data-i18n="play.subscribe_unlock")
       .alert.alert-danger#restricted-item-viewed.secret(data-i18n="inventory.restricted")
-    
-    button#choose-hero-button.btn.btn-lg.btn-primary.choose-inventory-active(data-i18n="play.change_hero") Change Hero
-    button#play-level-button.btn.btn-lg.btn-success.choose-inventory-active(data-i18n="common.play") Play
+    if !me.isStudent()
+      button#choose-hero-button.btn.btn-lg.btn-primary.choose-inventory-active(data-i18n="play.change_hero") Change Hero
+    if !view.options.hidePlay
+      button#play-level-button.btn.btn-lg.btn-success.choose-inventory-active(data-i18n="common.play") Play
```

Comment:
If the hidePlay tag is set in the InventoryModal, then they opened the InventoryModal through the chest icon instead of through a level.